packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -210,8 +210,8 @@ const controlStyles = (props: TProps) => (base: TBase, state: TState) => {
     })(),
     padding:
       props.appearance === 'quiet'
-        ? designTokens.paddingForInputAsQuiet
-        : designTokens.paddingForInput,
+        ? designTokens.spacing30
+        : designTokens.spacing30,
```

Comment:
Is this ok? Both conditions use the same value.

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/bdc818a94588d626b008a5a2a56681bad3e1f882

Thank you